postgres/init.sls

```diff
@@ -8,6 +8,11 @@ include:
 {{ postgres.conf_dir }}:
   file.directory:
     - makedirs: True
+    - user: postgres
+    - group: postgres
+    - dir_mode: 700
+    - clean: True
```

Comment:
It test's for `PG_VERSION` file. So if the base is initialized, it won't be removed. But if this dir contains something else, it would be cleared what is good. If you want to avoid this behaviour, add `postgres.clean_conf_dir` variable.